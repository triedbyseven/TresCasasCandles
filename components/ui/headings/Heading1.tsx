import React from 'react';
import { StyleSheet, Text } from "react-native";

export interface Heading1Props {
    text: string;
}
 
const Heading1: React.SFC<Heading1Props> = ({ text }) => {
    return <Text style={styles.text}>{text}</Text>;
};

const styles = StyleSheet.create({
    text: {
        fontSize: 40,
        color: '#263150',
        paddingLeft: 35,
        marginBottom: 25,
    }
});
 
export default Heading1;