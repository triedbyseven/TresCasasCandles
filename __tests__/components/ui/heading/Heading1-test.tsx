import 'react-native';
import React from 'react';
import Heading1 from '../../../../components/ui/headings/Heading1';
import { act } from 'react-test-renderer';

import renderer from 'react-test-renderer';

it('renders correctly', async () => {
    const result = renderer.create(<Heading1 text="Test String" />).toJSON();
    await act(async () => { expect(result).toMatchSnapshot() });
});
