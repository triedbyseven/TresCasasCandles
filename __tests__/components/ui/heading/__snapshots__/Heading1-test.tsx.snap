// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#263150",
      "fontSize": 40,
      "marginBottom": 25,
      "paddingLeft": 35,
    }
  }
>
  Test String
</Text>
`;
